azure-pipelines.yml

```diff
@@ -10,7 +10,9 @@ variables:
   buildConfiguration: 'Release'
   _SignType: real
   _TeamName: DotNetSpark
+  MSBUILDSINGLELOADCONTEXT: 1
   oldestCompatibleWorker: '0.9.0'
+  oldestCompatibleRelease: 'v0.9.0'
```

Comment:
oldestCompatibleWorker => backwardCompatibleRelease
oldestCompatibleRelease => forwardCompatibleRelease

Add comment that backwardCompatibleRelease/forwardCompatibleRelease is the "oldest" releases that work with the current release.

Reply:
remove `v` and add it when you need it.

Reply:
Done